specifyweb/statistics/views.py

```diff
@@ -0,0 +1,126 @@
+                                },
+                                'localityGeography': {
+                                    'type': 'object',
+                                    'properties': {
+                                        'countries': {
+                                            'type': 'integer'
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+            }
+        }
+    }}, )
+def collection_global(request) -> http.HttpResponse:
+    cursor = connection.cursor()
+    # Holdings
+    # Families
+    holding_dict = {}
+    cursor.execute("""
+    SELECT count(*) FROM (SELECT DISTINCT tx.TaxonID FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent = true) as Families1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Families1.TaxonID) as Families2, taxon as tx WHERE tx.rankid = 140 and Families2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber) as Familes3
+                  """, [request.specify_collection.id])
+    holding_dict['familiesRepresented'] = int(cursor.fetchone()[0])
+
+    # Genera represented
+    cursor.execute("""
+       SELECT count(DISTINCT tx.TaxonID) FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent <> 0) as Genera1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Genera1.TaxonID) as Genera2, taxon as tx WHERE tx.rankid = 180 and Genera2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber""",
+                   [request.specify_collection.id])
+    holding_dict['generaRepresented'] = int(cursor.fetchone()[0])
+
+    # UNQ_SPEC
+    cursor.execute("""
+       SELECT count(DISTINCT tx.TaxonID) FROM (SELECT DISTINCT tax.TaxonID,tax.nodenumber FROM (SELECT TaxonID FROM determination WHERE CollectionMemberID = %s AND determination.IsCurrent <> 0) as Genera1,taxon as tax WHERE tax.isaccepted <> 0 and tax.TaxonID = Genera1.TaxonID) as Genera2, taxon as tx WHERE tx.rankid = 220 and Genera2.nodenumber between tx.nodenumber and tx.HighestChildNodeNumber"""
+                   , [request.specify_collection.id])
+    holding_dict['speciesRepresented'] = int(cursor.fetchone()[0])
+
+    # PREP_BY_TYPE_LOTS
+    cursor.execute("""
+       SELECT pt.Name, trim(cast(concat(count(PreparationID), ' / ',  sum(countAmt)) as char(50))) FROM preparation p INNER JOIN preptype pt ON pt.PrepTypeID = p.PrepTypeID  WHERE CollectionMemberID = %s group by pt.Name""",
+                   [request.specify_collection.id])
+    prepbytypelots_result = cursor.fetchall()
+
+    preptypelotstotal_dict = {}
+    for preptype_stat in list(prepbytypelots_result):
+        lots_total_arr = preptype_stat[1].split(' / ')

```

Comment:
This is redundant. Why do you contac it in the query and then split it here? Instead, make your query return the value as two columns